docs/release.md

```diff
@@ -0,0 +1,18 @@
+# Release
```

Comment:
BTW on the release stuff it might make sense to suggest using github's release system. I can even add something like that in the docs with a guide for how to use gh cli to create release versions. Secondly, this repo lacks a contributor.md file this might be a good idea.

Reply:
I don't know anything about Github's release system, so I can't comment on whether we should use it for Enduro.

I agree that a `contributor.md` would be a good addition.

Reply:
I will use a temporary contributor md and then add an issue to work on it later